SKILL.md

```
@@ -1,47 +1,166 @@
-
-Testing wallets should be used for [automated testing](./references/automated-testing.md).
-
-It is recommended to write tests so that the agent can test its own work and fix bugs itself without requiring human input.
-
-## Production Wallet
-
-If they do not have a wallet yet [here are some options](./references/production-wallets.md)
\ No newline at end of file
+User → says anything natural → agent does everything → confirmation.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)